js_modules/dagit/packages/core/src/ui/Colors.tsx

```diff
@@ -0,0 +1,40 @@
+export const ColorsWIP = {
+  Dark: 'rgba(23, 22, 21, 1)',
+  White: 'rgba(255, 255, 255, 1)',
+  Gray900: 'rgba(35, 31, 27, 1)',
+  Gray800: 'rgba(58, 54, 49, 1)',
+  Gray700: 'rgba(82, 78, 72, 1)',
+  Gray600: 'rgba(107, 103, 98, 1)',
+  Gray500: 'rgba(134, 131, 127, 1)',
+  Gray400: 'rgba(161, 157, 153, 1)',
+  Gray300: 'rgba(189, 186, 183, 1)',
+  Gray200: 'rgba(218, 216, 214, 1)',
+  Gray100: 'rgba(245, 244, 242, 1)',
+  Gray50: 'rgba(250, 249, 247, 1)',
+  Gray10: 'rgba(35, 31, 27, 0.03)',
+  Blue1: 'rgba(14, 12, 167, 1)',
+  Blue2: 'rgba(79, 67, 221, 1)',
+  Blue3: 'rgba(185, 180, 241, 1)',
+  Blue4: 'rgba(14, 12, 167, 0.08)',
```

Comment:
Just a stupid nit but it'd be nice to keep these all fully opaque so you can put a Blue4 button inside a Gray900 box (or something like that hypothetically). I think there's a website that we can paste these into and have them all output as opaque colors, so we can try it later if this ever comes up!

Reply:
The opacity here is by specific request from B&B, but let's keep an eye on it and see if it becomes an issue.